packages/components/src/TCRUserDashboard/DashboardStyledComponents.tsx

```diff
@@ -352,6 +352,7 @@ export const StyledTransferTokenTitle = styled.div`
   text-align: center;
   margin: 0 0 40px;
   padding: 20px;
+  flex-grow: 1;
```

Comment:
Just to confirm, are these guys constrained to any max-width anywhere or by their parent? Because on very wide screen full width looks a bit wack.

Reply:
I'll add a max-width to the whole thing eventually, but we already had some things that weren't constrained at all and it was way worse when they were different sizes. At least now they're all the same size